pkg/cloudprovider/aws/fake/ec2api.go

```diff
@@ -181,6 +181,57 @@ func (e *EC2API) DescribeInstanceTypesPagesWithContext(ctx context.Context, inpu
 					Ipv4AddressesPerInterface: aws.Int64(60),
 				},
 			},
+			{
+				InstanceType:                  aws.String("p3.8xlarge"),
+				SupportedUsageClasses:         []*string{aws.String("on-demand")},
+				SupportedVirtualizationTypes:  []*string{aws.String("hvm")},
+				BurstablePerformanceSupported: aws.Bool(false),
+				BareMetal:                     aws.Bool(false),
+				ProcessorInfo: &ec2.ProcessorInfo{
+					SupportedArchitectures: aws.StringSlice([]string{"x86_64"}),
+				},
+				VCpuInfo: &ec2.VCpuInfo{
+					DefaultVCpus: aws.Int64(32),
+				},
+				MemoryInfo: &ec2.MemoryInfo{
+					SizeInMiB: aws.Int64(249856),
+				},
+				GpuInfo: &ec2.GpuInfo{
+					Gpus: []*ec2.GpuDeviceInfo{{
+						Manufacturer: aws.String("NVIDIA"),
+						Count:        aws.Int64(4),
+					}},
+				},
```

Comment:
Do you need to include GPU Memory here as well?

Reply:
In general, you're right, but k8s isn't mature enough to specify GPU memory yet.